### ISSUE TITLE
Feature request: Distraction free mode

### ISSUE THREAD

**Issue by armundle** [HappenApps/Quiver#561]
It would be great to have a distraction free mode, something along the lines of [iA Writer ](https://ia.net/writer/mac/) minimal view (or the preview mode itself).

**Comment by s303** [comment]
Even better would be have this mode for a cell.

**Comment by daneverson** [comment]
I agree with @armundle, if the preview mode were editable it would make a great distraction-free writing mode.

If this were to make it to your roadmap, I'd also ask you to consider "typewriter scrolling", as used in Ulysses and some other editors. I think for any kind of prose writing, this is incredibly helpful for focus since it prevents having to do goofy things like add a bunch of blank lines at the bottom of the document, or always stare at the bottom 5% of your screen.
